src/disk.c

```diff
@@ -510,6 +500,15 @@ static int disk_read (void)
 		else
 			ssnprintf (disk_name, sizeof (disk_name), "%i-%i", disk_major, disk_minor);
 
+		DEBUG ("disk plugin: disk_name = \"%s\"", disk_name);
+
+		/* check the name against ignore list */
+		if (ignorelist_match (ignorelist, disk_name) != 0) {
```

Comment:
There was no these changes in first PR, so I decided to inform you about new changes appeared.